### PR TITLE
Map+Markers: Add timezone overlay and timestamps

### PR DESCRIPTION
**Pull request by networkException**
This pull request adds an overlay with the outlines of timezones

image

As well as timestamps to the markers on the map

image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,14 +80,21 @@
...
+
           for (const user of data) {
             const container = document.createElement("div");
             const heading = document.createElement("h1");
             heading.innerText = user.nick;
 
             container.appendChild(heading);
 
-            for (const [ label, link ] of Object.entries(user.links)) {
+            for (const [label, link] of Object.entries(user.links)) {
```
Comment (linusg) [inline index.html:97]: please put the style fix in the previous commit instead :)

File: index.html [hunk 2]
```diff
@@ -95,7 +107,49 @@
             const icon = new L.Icon.Default();
             icon.options.className = user.contributor ? "contributor" : "";
 
-            L.marker(user.coordinates, { icon }).addTo(map).bindPopup(container);
+            L.marker(user.coordinates, { icon })
+              .addTo(map)
+              .bindPopup(() => {
+                (() => {
```
Comment (linusg) [inline index.html:113]: does this need to be an IIFE?

**Review by linusg — COMMENTED** [review]
love it!

File: index.html (hunk 2 above)
Comment (networkException) [reply index.html]: Did some lazy refactor and didnt notice I'm already in a nested scope :^)

**Pull request merged**